### PR TITLE
Add domain to translatable strings in Classic template files

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add domain to all translatable strings **in template files** |
| Type? | new feature |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Ask @jtabet and @Shine-neko |

Ping @AlexEven 
### Codemod

```
codemod -m -d . --extensions tpl "(\{l\ss='[^']+')" "\1 d='Shop.'"
```

### CODE REVIEW

File: themes/classic/templates/_partials/form-field.tpl [hunk 1]
```diff
@@ -15,7 +15,7 @@
       {if $field.type === 'select'}
 
         <select class="form-control form-control-select" name="{$field.name}" {if $field.required}required{/if}>
-          <option value disabled selected>{l s='-- please choose --'}</option>
+          <option value disabled selected>{l s='-- please choose--' d='Shop.Forms.Labels'}</option>
```
Comment (maximebiloe) [inline themes/classic/templates/_partials/form-field.tpl]: I think the space removed is an error, no?

File: themes/classic/templates/checkout/_partials/steps/personal-information.tpl [hunk 2]
```diff
@@ -4,19 +4,19 @@
   {if $customer.is_logged && !$customer.is_guest}
 
     <p class="identity">
-      {l s='Connected as [1]%1$s %2$s[/1].'
+      {l s='Connected as [1]%1$s %2$s[/1].' d='Shop.Theme.CustomerAccount'
```
Comment (maximebiloe) [inline themes/classic/templates/checkout/_partials/steps/personal-information.tpl]: Can you put the `d` parameter on a new line, please? Also, put the closing brace on a new line too?

File: themes/classic/templates/customer/order-detail.tpl [hunk 3]
```diff
@@ -41,12 +41,12 @@
 
   {block name='order_history'}
     <section id="order-history" class="box">
-      <h3>{l s='Follow your order\'s status step-by-step'}</h3>
+      <h3>{l s='Follow your order\' d='Shop.Theme.CustomerAccount's status step-by-step'}</h3>
```
Comment (maximebiloe) [inline themes/classic/templates/customer/order-detail.tpl]: I think something went wrong here ^^

**Comment by kpodemski** [comment]
What is `d` param? It is **context**?

**Comment by julienbourdeau** [comment]
It's the _domain_
We're moving to a more standard system: http://api.symfony.com/3.0/Symfony/Component/Translation/TranslatorInterface.html#method_trans

**Comment by kpodemski** [comment]
I'm not sure what `domain` means when it comes to translations, it is more like `context` so i'm confused with this naming convention, that's why I'm asking :D

**Comment by julienbourdeau** [comment]
@AlexEven is writing an article about it. But yeah, it's some sort of context for your string.

**Comment by kpodemski** [comment]
@julienbourdeau so it will work as I suggested, right? If so, super nice.

https://trello.com/c/YyylcAR3/273-global-language-variables

**Pull request merged**